packages/office-ui-fabric-react/src/components/Modal/Modal.types.ts

```diff
@@ -111,10 +118,13 @@ export type IModalStyleProps = Required<Pick<IModalProps, 'theme'>> &
     hasBeenOpened?: boolean;
     /** Positioning of modal on first render */
     modalRectangleTop?: number;
+    /** Whether the modal is sticky (can interact with content behind the dialog without dismissal) */
+    isSticky?: boolean;
```

Comment:
I would move this to the `Pick` on line 112 to avoid redundant definitions that can get out of sync. (looks like a few of the others should be moved too)

Reply:
i.e.
```tsx
Pick<IModalProps, 'className' | 'containerClassName' | 'scrollableContentClassName' | 'topOffsetFixed' | 'isSticky'> 
```

Reply:
I updated mine but didn't change any of the other ones, like isOpen, because it is the name of a prop but what's really being passed is the isOpen state